File: app/src/main/java/dev/pitlor/smssync/viewmodels/ConversationsListItemViewModel.java
package dev.pitlor.smssync.viewmodels;

import android.app.Application;
import android.view.View;
import android.widget.ImageView;

import androidx.annotation.NonNull;
import androidx.databinding.BindingAdapter;

import com.squareup.picasso.Picasso;

import java.io.File;

import dev.pitlor.sms.Message;
import lombok.Data;
import lombok.Getter;
import lombok.Setter;

public class ConversationsListItemViewModel extends BaseViewModel {
    @Setter public Message message;

    public ConversationsListItemViewModel(@NonNull Application application) {
        super(application);
    }
}

File: app/src/main/java/dev/pitlor/smssync/viewmodels/ConversationsListItemViewModel.kt
package dev.pitlor.smssync.viewmodels

import android.app.Application
import androidx.hilt.lifecycle.ViewModelInject
import dev.pitlor.smssync.datasources.Message

class ConversationsListItemViewModel @ViewModelInject constructor(application: Application) : BaseViewModel(application) {
    var message: Message? = null
}